src/views/HomeView.vue

```diff
@@ -60,19 +60,27 @@ const eventRows = computed(() => {
         firstTeam: {
           name: teamOneInfo.name,
           id: teamOneInfo.id,
-          spread: { id: teamOneSpreadId, 
-                    value: teamOneSpread },
-          ml: { id: teamOneMLId,
-                value: teamOneML } 
+          spread: {
+             id: teamOneSpreadId, 
+             value: teamOneSpread
+             },
+          ml: {
+            id: teamOneMLId,
+            value: teamOneML
+            } 
         },
         secondTeam: {
           name: teamTwoInfo.name,
           id: teamTwoInfo.id,
-          spread: { id: teamTwoSpreadId, 
-                    value: teamTwoSpread },
-          ml: { id: teamTwoMLId,
-                value: teamTwoML } 
-          },
+          spread: {
+            id: teamTwoSpreadId, 
+            value: teamTwoSpread 
+            },
+          ml: { 
+            id: teamTwoMLId,
+            value: teamTwoML
+            } 
+        }
```

Comment:
Spaces vs tabs here.

Also, line 66, 70, 78, and 82 are all indented when they should be on an equal indentation to the level they're closing.

Reply:
Converted all whitespace to tabs, unindented lines 66, 70, 78, 82